src/pnpq/devices/polarization_controller_thorlabs_mpc320.py

```diff
@@ -30,11 +30,11 @@
 
 
 class PolarizationControllerParams(TypedDict):
-    velocity: int
+    velocity: Quantity
```

Comment:
Oh yes, that's a really good idea. But as we increase the amount of `Quantity`s, I feel like it would be more type-safe if we had a class for each one, like we discussed that one time at KBIC. Then we can do rounding in each of those as well.

Reply:
Okay, yes. That makes sense. If the pull request to add generics is still open for over a year in the pint github, it probably isn't a task for us...

And yeah, that makes sense. We don't really have a use for the "real" value. I am just sad to see information disappear.